src/upgrades/mod.rs

```diff
@@ -160,7 +158,17 @@ pub trait UpgradeStore {
 
 /// Should be called when Krill starts, before the KrillServer is initiated
 pub fn pre_start_upgrade(config: Arc<Config>) -> Result<(), UpgradeError> {
-    upgrade_0_9_0(config)
+    upgrade_data_to_0_9_0(config)
+}
+
+/// Should be called when Krill starts, before the KrillServer is initiated
```

Comment:
This comment is the same as the comment on `pre_start_upgrade`, is that intentional?